### DISCUSSION TITLE
Are some functions not documented intentionally?

### DISCUSSION THREAD

**Discussion by ByScripts** [vueuse/vueuse#2885] [Q&A] [answered]
Hi,

I stumbled upon a few useful functions by chance, and I wonder if there is a reason they are not documented.

For example, the `promiseTimeout`.

Is it intentional? Can they be used safely?

Thank you.

**Comment by ddenev** [comment]
@antfu , I wander the same thing. How many other useful functions are not documented?

**Comment by antfu** [comment, marked as answer]
They are not technically composables but they are some utilities we need for creating some composables. If you want to help, maybe we could have a dedicated page for all those utilities functions we expect to exposed to users.
